setup.py

```diff
@@ -78,7 +78,7 @@ def run(self):
     "pydot>=1.2.4",
     "speedcopy>=2.0.1; python_version < '3.8' and sys_platform == 'win32'",
     "flatten_json>=0.1.6",
-    "texttable>=0.5.2",
+    "tabulate>=0.8.7",
```

Comment:
This does not support aligning headers. It does have `colalign` but it aligns the whole column, and, I don't think it's a deal-breaker.